Skip polar lats in validation random point selection; expand validation docs (#617)
dynamical-org/reformatters: docs/validation.md

**Opening sentences (always).** Begin with one or two sentences stating where the dataset stands. In an early draft this might just be "Initial validation pass; see `### For further review` for open items." In a final draft and in published reports it should affirm that the dataset has been reviewed and is ready for use, and briefly call out anything from `### Review notes` that needs special care from a user (ideally there is nothing and the sentence is just "This dataset has been reviewed and is ready for use.").

**Subsections (up to three, included as needed).** Each is a bulleted list.

- **`### For further review`** — issues an expert dataset creator should look into before the report is published, each with a link to the image(s) where the issue is apparent. Audience is the internal reviewer. **Must be removed from final-draft and published reports** — any item that ends up worth surfacing to users belongs under `### Review notes` instead.

> - **`### What looks good`** — short, positive summary confirming the checks in [section 4](#4-data-quality-checklist) passed. These have generally been coming out clean; keep this section brief and don't duplicate detail already covered by the doc.

Author: ```suggestion
- **`### What looks good`** — short, positive summary confirming the checks in [section 4](#4-data-quality-checklist) that passed. Keep this section brief and don't duplicate detail already covered by the doc.
```

- [ ] **Reference availability.** If the reference dataset doesn't cover your window, `validation_summary.md` will show `variable not available in reference dataset` — that is not a bug in the validation dataset, but spatial / temporal comparisons lose their signal. Consider a different reference or re-run on a time range the reference covers.
- [ ] **Ensemble member is plausible.** For ensemble datasets `validation_summary.md` records the randomly-selected member. Rerun once more to confirm that a different member also looks right.

## 5. Sharing and publishing the report

Once a run is reviewed, render it to a static HTML report, share it as one or more drafts for internal and external review, and finally publish the approved version. Two storage paths exist:

- **Draft** — every non-final upload goes here, timestamped, kept forever. Use for sharing a single run for review without committing to it.

> - **Stable** — the canonical, published report for a dataset, overwritten by each new publish. Embedded in the dynamical-stac catalog and linked from dynamical.org.

Author: ```suggestion
- **Stable** — the canonical, published report for a dataset, overwritten by each new publish. Embedded in dynamical.org.
```

**Phase 2 — External-audience draft.** Audience: external dataset users who have never seen the run directory or our review process. As soon as `### For further review` is empty, rewrite the `## Summary` block for that audience and upload one more draft (still no `--publish`):

- Update the opening sentences to affirm the dataset is reviewed and ready for use, and to call out anything from `### Review notes` that needs special care.
- Drop the (now empty) `### For further review` subsection.
- Reword every remaining bullet for a public dataset consumer: spell out variable names, expand acronyms, avoid `P1`/`P2` (use the explicit lat/lon or describe the point), avoid ticket numbers, internal codenames, file paths, and process shorthand. Each bullet must make sense in isolation to someone with no repo context.

**Phase 3 — Publish.** An expert reviewer reads the Phase 2 draft and gives the go-ahead. Only then run `upload <run-dir> --publish` to write to the stable path. Do not run `--publish` while `### For further review` is non-empty or while the summary still reads as internal-audience prose — share another draft instead.

> `upload` re-renders the HTML before uploading, so editing `validation_summary.md` and re-running `upload` is the full iteration loop; no separate `render-report` step is needed.

Author: Remove this line (and the line it was replacing). We no longer render to html in here just publish the .md

2. The STAC generator surfaces it as an asset with role `validation-report` (type `text/html`).
3. Run `./scripts/generate` and commit `stac/`.

> This is a one-time wiring per dataset. Subsequent `upload --publish` runs update the report contents at the same URL — no STAC change needed.

Author: This whole section is out of date. Instead we just redeploy the dynamical.org site and its build will pull the latest published report for each dataset and incorporate it into the catalog documentation for each data product.
